tests/utils.go

```diff
@@ -1458,25 +1450,6 @@ func GetVmPodName(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance
 	return podName
 }
 
-func AppendEmptyDisk(vmi *v1.VirtualMachineInstance, diskName string, busName v1.DiskBus, diskSize string) {
```

Comment:
Deleting dead code is noble. But good commits do one thing. I'd put it in a commit of its own.

Reply:
Done